lib/groonga/groonga-log.rb

```diff
@@ -0,0 +1,8 @@
+require "groonga/groonga-log/version"
+require "groonga/groonga-log/parser"
+
+module Groonga
```

Comment:
We don't need `Groonga` namespace.

Reply:
I've followed groonga-query-log gem. It uses `Groonga` namespace.

Reply:
groonga-query-log should be changed too.